lo_2010.cpp

```diff
@@ -22,8 +22,6 @@ ncmt=1, trans=11
 D_CENT = 1; 
 double NORM_WT = 70;
 double NORM_PMA = 30;
-double Fsga = ?;
-double spiro = ?;
 double CLi = 1.0 * pow((wt/NORM_WT), 0.75) * pow((pma/NORM_PMA), 3.16) * Fsga
```

Comment:
ok so looking in the actual publication, this F term is misleading. I originally thought it was a 1/0, but that wouldn't makes sense as if the baby wasn't small for gestantioal age, CLi would be 0. Looking into the actual publication, we see that the full CL equation was actually

```
CL = 1.0 * (wt/70)^0.75*(PMA/30)^3.16*[0.83*SGA + 1.03*(1-SGA)]
```

so we should expand this equation to just have the parameter be called SGA (with a default of 0) and then use this equation.